docusaurus/docs/engineering/knowledge-base/development-lifecycle.md

```diff
@@ -4,154 +4,230 @@ sidebar_position: 7
 
 # Development Lifecycle
 
-To look at/download BobaBoard's code, check out our guide to [Setting Up your Coding Environment](/docs/engineering/start-developing/before-you-start).
-You might also be interested in [Git Notes](/docs/engineering/legacy-pages/git-notes)
+If you're new to collaborating on a code project, you may be wondering what that process looks like. This page gives an overview of the development lifecycle, explains key terminology, and guides you through the step-by-step process of contributing to BobaBoard's codebase.
 
-:::tip This seems scarier than it is!
-If you're new to coding and github, the following process might seem scary and overwhelming. Don't worry: it will become second nature very quickly! If you want someone to hold your hand throughout the process, contact Ms. Boba.
-:::
+To slowly get you familiar with more technical vocabulary you might run into, this document also includes relevant information about git and GitHub. For more information on these subjects you can look at our [Git Notes](/docs/engineering/legacy-pages/git-notes.md) page.
 
 :::note
-TODO[Ms.Boba]: Explore whether the new github command line interface can make this process less cumbersome.
+We will use GitHub as our example platform since BobaBoard's code is hosted there. However, there are alternative platforms that serve similar purposes.
 :::
 
-## Development lifecycle overview
+As a side note, if you've looked at our [Recommended Tools](/docs/engineering/knowledge-base/recommended-tools.md) document, you might be aware that we recommend Visual Studio Code as a code editor. One reason for this is its excellent integration with git. This lets you perform various tasks using the editor's user interface instead of typing commands in your terminal. Feel free to use integrations if you prefer. However, we will still list the commands here so that everyone can follow along and understand what's happening behind the scenes.
 
-At a high level, the git development lifecycle follows these steps:
+## Overview
 
-0. **The developer "forks" the code.** This creates a separate copy of the codebase owned by the developer itself. No matter what changes the developer makes, the original codebase (called `upstream`) will remain unchanged.
-1. **The developer creates a "feature branch" on their "fork".** When the developer first forks the code, they are placed on the official "current version" of the codebase, called `master` (or `main`). Rather than modify `master` directly, the developer then creates a "feature branch" to make their edits on.
+So, you've decided to contribute to a project. What happens next?
 
-   For example, if the developer is working on adding a "read more" feature, they might create a branch called `read-more`. Creating branches allows a developer to work on many independent features at the same time.
+1. **Add a copy of the project's code to your own GitHub account.**
```

Comment:
To make the numbering work correctly, you'll have to indent the text underneath it with the same amount of characters as the list numbers. Something like:
```
1. **Add a copy of the project's code to your own GitHub account.**
   >  This first step involves getting your own version of the project's code to edit to your heart's content. In technical terms, this means creating a 'fork' of the project you want to contribute to.
```

Note that the `>` sits at the same indentation as the first word of the list item. I might be mistaken on the exact number of characters, but it should work.

Same below.